components/composer/src/Composer.svelte

```diff
@@ -143,6 +141,18 @@
     focus_body_onload: false,
   };
 
+  let message: Message = JSON.parse(JSON.stringify(messageInitialState));
```

Comment:
So `message` no need to be a store. And move the `attachments` store inside the Composer component.